pkg/spec/postgresql.go

```diff
@@ -165,9 +170,13 @@ func (m *MaintenanceWindow) UnmarshalJSON(data []byte) error {
 		return fmt.Errorf("'From' time must be prior to the 'To' time")
 	}
 
+	if (int(got.StartWeekday)+6)%7 > (int(got.EndWeekday)+6)%7 {
+		return fmt.Errorf("'From' weekday must be prior to the 'To' weekday")
```

Comment:
Got it; don't we need to check the weekdayProvidedTo (otherwise, got.StartWeekday == got.EndWeekday == 0)